resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/YarnAllocator.scala

```diff
@@ -288,9 +296,14 @@ private[yarn] class YarnAllocator(
       s"executorsStarting: ${numExecutorsStarting.get}")
 
     if (missing > 0) {
-      logInfo(s"Will request $missing executor container(s), each with " +
-        s"${resource.getVirtualCores} core(s) and " +
-        s"${resource.getMemory} MB memory (including $memoryOverhead MB of overhead)")
+      var requestContainerMessage = s"Will request $missing executor container(s), each with " +
```

Comment:
Minor, but this now should be inside a `if (log.isInfoEnabled())` block.

Reply:
Fair enough, added a condition to check whether it's enabled.